xcode/Subconscious/Shared/Components/Common/MarkupTextViewRepresentable.swift

```diff
@@ -123,16 +123,43 @@ struct MarkupTextViewRepresentable: UIViewRepresentable {
                 )
             )
         }
+        
+        // This allows us to intercept touches on embedded transclude blocks
+        override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
```

Comment:
Just realized we need to be calling super for every logical branch that is failing or not meant to interecpt a transclude tap. That way we fall back to default touchesBegan behavior upon failure.

Perhaps we should remove this logic in this PR, since it is intended to be a no-op for now anyway.

Reply:
Removed this code, we'll add it back when we need it 👍